libs/blocks/marquee/marquee.css

```diff
@@ -203,6 +203,20 @@
   object-position: left bottom;
 }
 
+.marquee .text .icon-area.icon-area-multiple {
+  display: flex;
+}
+
+.marquee .text .icon-area.icon-area-multiple img {
```

Comment:
Same here:

```suggestion
.marquee .icon-area-multiple img {
```

Reply:
I simplified it to:

`.marquee .icon-area.icon-area-multiple {`

because the styles from `marquee.split .icon-area {` or `marquee.large .icon-area {` are overriding this (even if the css is ordered above this one) so this won't work without putting `.icon-area` next to `.icon-area-multiple`